<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LogParser v0.1</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../Téléchargements/ZpKyf3ZBgLFQTpMM1OeLzCjBV8xta9O0_0/logparser.jpg</normaloff>../../../../Téléchargements/ZpKyf3ZBgLFQTpMM1OeLzCjBV8xta9O0_0/logparser.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnParseLogs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Parse Logs</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_err404">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>341</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_read">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>2</y>
      <width>511</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_save">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>0</y>
      <width>97</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Save report</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_err404">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Erreurs 404</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_GoogleBot">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>241</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Pages Crawlées par GoogleBot</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_GoogleBot">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>371</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>739</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
